scss/components/_dropdowns.scss

```diff
@@ -52,6 +52,10 @@
     padding: 1rem .5rem 1rem 1.5rem;
     right: .5rem;
   }
+
+  & + .dropdown__panel {
```

Comment:
Yeah, this looks like a bug in linter... unless you can start a nested rule with `+ .class`????